src/ape/exceptions.py

```diff
@@ -1,4 +1,9 @@
-from typing import Optional
+from typing import TYPE_CHECKING, Optional
+
+from eth_utils import humanize_hash
```

Comment:
hmm... probably could use this a lot more in the codebase

Reply:
Yeah, TIL